.gitmodules

```diff
@@ -3,7 +3,7 @@
 	url = git://github.com/galaxyproject/ansible-galaxy-tools.git
 [submodule "roles/galaxyprojectdotorg.galaxy-extras"]
 	path = roles/galaxyprojectdotorg.galaxy-extras
-	url = git://github.com/galaxyproject/ansible-galaxy-extras
+	url = git://github.com/mvdbeek/ansible-galaxy-extras
```

Comment:
Ok this I knew it actually and I saw the changes for ftp working, it's good. Actually it was not quit clear in my mind except that something was somewhat hurting my sense of "sustainability". The question arises of what happens if a submodule evolves differently ? I guess you clone a given precise commit ? but what happens if a submodule is just deleted ? say the developper dies :-) ?

Would it be possible instead to fork a submodule and to continue to use this forked submodule as a submodule ?